task3/src/models/index.js

```diff
@@ -0,0 +1,23 @@
+import Sequelize from 'sequelize';
+
+import sequelize from '../data-access';
+import UserModel from './user';
+
+const models = {
+    UserModel: UserModel.init(sequelize, Sequelize)
+};
+
+// unnecessary as we don't have assosiations for now, but I'll keep it for further purposes
+Object.values(models)
+    .filter((model) => typeof model.associate === 'function')
+    .forEach((model) => model.associate(models));
+
+// uncomment to recreate Users table
+// models.UserModel.sync({ force: true });
+
+const db = {
```

Comment:
Why do we need to group it together?

Reply:
Probably we do, at least in current implementation where I defined static method init in model, so we hav to return already initialized model either in each model or here.( Which we don't in model) So, when I have several models I'd iterate them through loop and init each of them here.

Reply:
I've changed my mind and I'm going to init and associate models in each file